src/gui/editors/AutomationEditor.cpp

```diff
@@ -1510,8 +1510,12 @@ void AutomationEditor::paintEvent(QPaintEvent * pe )
 		case SELECT: cursor = s_toolSelect; break;
 		case MOVE: cursor = s_toolMove; break;
 	}
-	p.drawPixmap( mapFromGlobal( QCursor::pos() ) + QPoint( 8, 8 ),
-								*cursor );
+	// TODO RON copy all if, replace the top left etc
```

Comment:
Did you forget to remove this?

Reply:
so sorry, totally forgot that line there.